Item.py

```diff
@@ -12,14 +12,17 @@ def __init__(self, name='', event=False):
         if event:
             type = 'Event'
             progressive = True
+            repeatable = None
             itemID = None
             special = None
         else:
-            (type, progressive, itemID, special) = item_table[name]
+            (type, progressive, repeatable, itemID, special) = item_table[name]
 
         self.name = name
         self.advancement = (progressive == True)
         self.priority = (progressive == False)
+        self.repeatable = (repeatable == (True or False))
+        self.onetime = (repeatable == (None or False))
```

Comment:
I don't think these lines do what you want. `(True or False)` evaluates to `True`, so this is the same as `repeatable == True`. `None or False` evaluates to `False`. Maybe you want `repeatable in (True, False)`, to test that repeatable is one of those options?